### DISCUSSION TITLE
Express returns null - simple demo

### DISCUSSION THREAD

**Discussion by pcwSlide** [deta/cloud-docs#745] [Help] [answered]
following : https://docs.deta.sh/docs/micros/getting_started/#creating-your-first-micro

Simple node/express 

My URL: https://ai8rji.deta.dev/

Returns: Null

It seems simple - you are doing all the back end run scripts?
Why?

**Comment by pcwSlide** [comment, marked as answer]
Ok - update dependencies..

**Reply by pcwSlide** [reply]
This can be closed - can I do this?

**Reply by abdelhai** [reply]
this is a forum and questions are meant to be there to assist users when they face the same issue in the future
